aspnetcore/signalr/authn-and-authz.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Authentication and Authorization in SignalR
+author: rachelappel
+description: Learn how to use authentication and authorization in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 05/01/2018
+uid: signalr/authn-and-authz
+---
+
+# Authentication and authorization in SignalR
+
+By [Andrew Stanton-Nurse](https://twitter.com/anurse)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/authn-and-authz/sample/ ) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Authenticate users connecting to a SignalR Hub
+
+SignalR can be used with [ASP.NET Core Authentication](xref:security/authentication/index) to associate a User with each connection. In a Hub, authentication data can be accessed from the [`HubConnectionContext.User`](/dotnet/api/microsoft.aspnetcore.signalr.hubconnectioncontext.user?view=aspnetcore-2.1) property. Once a connection is authenticated, it is possible to invoke methods on all connections associated with a specific user. See [Manage users and groups in SignalR](xref:signalr/groups) for more information.
+
+### Cookie Authentication
+
+In a simple web application, cookie authentication allows your existing user credentials to automatically flow to SignalR connections. When using the browser client, no additional configuration is needed. If the user is logged in to your application, the SignalR connection will automatically inherit this authentication.
```

Comment:
simple -> is that necessary? and not true for more complex apps?
application -> app
will automatically inherit -> automatically inherits

Reply:
I'll clarify what is meant by simple. We generally do not recommend using cookies in anything that may have multiple clients (i.e. not just the browser)